io/net/bonding.py

```diff
@@ -579,4 +585,5 @@ def error_check(self):
             self.fail("Tests failed. Details:\n%s" % "\n".join(self.err))
 
     def tearDown(self):
-        self.session.quit()
+        if self.host_interfaces:
+            self.session.quit()
```

Comment:
why this change is this was bug earlier , if yes push this in another commit

Reply:
this is to safely exit  teardown.. if interfaces are not available.. or wrong user input